README.md

```diff
@@ -56,23 +69,26 @@ $ make test
 
 ### Running the services
 
-The first time you run these services, you'll need to upload your policy:
-```console
-make update-policy
-```
-You only need to do this once. (If you make changes to the policy, you'll need to run it again.)
-
-Run GitClub in one terminal:
-
-```console
-make -C services/gitclub
-```
-
-Run the Jobs Service in another terminal:
-
-```console
-make -C services/jobs
-```
+1. The first time you run these services, you'll need to upload your policy:
+    ```console
+    make update-policy
```

Comment:
If we're stating that `make` is not required (only recommended), perhaps we could provide alternative instructions here (if the user doesn't have `make`)?

It might read something like,
> If you don't have `make` installed, you'll need to execute `oso-cloud policy policy/authorization.polar`.

Reply:
Yeah that seems like too much. Maybe we'll just say make is required for the build process.